origin-growth/src/resources/invite.js

```diff
@@ -158,17 +170,13 @@ class GrowthInvite {
     // We need to compute pending invites only if the campaign is active.
     let pendingInvites = []
     const now = new Date()
-    const isActive = campaign.startDate >= now && campaign.endDate <= now
+    const isActive = campaign.startDate <= now && campaign.endDate >= now
```

Comment:
ahah. good catch ! thanks for fixing this bug in my code :)

Reply:
👍 
after tests pass will merge directly